vsintegration/src/FSharp.Editor/LanguageService.fs

```diff
@@ -49,14 +55,42 @@ type internal SVsSettingsPersistenceManager = class end
 type internal FSharpLanguageService(package : FSharpPackage) =
     inherit AbstractLanguageService<FSharpPackage, FSharpLanguageService>(package)
 
+    // The latest component model.  This is used to retrieve IDiagnosticAnalyzerService.
+    // FSROSLYNTODO: Work out how to make this and the two tables below non-static.
+    static let mutable componentModelOpt : IComponentModel option = None
```

Comment:
I'd like someone who understands MEF to go through and remove all these statics please.  I don't think adding this one causes any additional problems that aren't already there, but it's still not right.  I don't understand anything much about MEF.